.github/workflows/cd.yml

```diff
@@ -56,6 +75,18 @@ jobs:
     runs-on: self-hosted
     needs: build
     steps:
+      - name: On staging env
+        env:
+          REMOTE_URL: developer-example.staging.cosmian.com
+          DOCKER_COMPOSE_FILE: saas_applications_examples/docker-compose.yml
+        run: |
+          ssh cosmian@${REMOTE_URL} docker login ${{ env.REGISTRY }} -u ${{ github.actor }} -p "${{ secrets.GITHUB_TOKEN }}"
+          ssh cosmian@${REMOTE_URL} docker-compose -f ${DOCKER_COMPOSE_FILE} pull
+          ssh cosmian@${REMOTE_URL} docker-compose -f ${DOCKER_COMPOSE_FILE} -p saas_applications_examples_staging down
+          ssh cosmian@${REMOTE_URL} docker-compose -f ${DOCKER_COMPOSE_FILE} -p saas_applications_examples_staging up -d
+          ssh cosmian@${REMOTE_URL} service nginx status
+          ssh cosmian@${REMOTE_URL} docker system prune -f
+
       - name: On prod env
         env:
           REMOTE_URL: developer-example.cosmian.com
```

Comment:
On the path jsut under - same : it should be : saas_applications_examples/prod/docker-compose.yml